WordPressKit/WordPressKit/WordPressKitConstants.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+
+/// Constants definition for WordPressKit
+struct WordPressKitConstants {
+    struct SiteSubscription {
+        struct Delivery {
+            static let frequency = "delivery_frequency"
+        }
+    }
+}
```

Comment:
@jleandroperez I created this with the thought to add in future all the constants used within WordPressKit. The nested structures are only to divide the constants domain. Can i just get rid of `SiteSubscription` and leave `WordPressKitConstants` rather than use an extension of `ReaderTopicServiceRemote`?

Reply:
Reasons why i was suggesting to move this constant as a nested struct are: 

- This particular constant isn't used anywhere else, _but_ in the `ReaderTopicServiceRemote`
- If we were to implement WordPressKitConstants, it would probably make more sense to actually move all of the constants there. Otherwise we have different code styles / approaches.
- (And finally!) having a shared constants collection does make sense, as long as there is reuse. If there isn't... personally, i find there is nothing more practical than having the definitions where you need them. That is, unless the container size goes off hands!!